Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -0,0 +1,121 @@
+:root {
+    /*colors*/
+    --primary: #0C6594;
+    --highlight: #DEF3FE;
+    /*not needed?*/
+    --gray: #C4C4C4;
+    /*not needed?*/
+    --lightgray: #E1E1E1;
+    --menubackground: #F9F9F9;
+    --hover: #218FC9;
+    --border: #979797;
+    --yellow: #DFAF00;
+    --red: #DA2222;
+    --green: #4AD439;
+    --focus: #23ADF8;
+    --header: #F2F2F2;
+
+    --dropdown-scrollbar-track: #E3E3E3;
+    --scrollbar-track: #D3D3D3D3;
+    --background: #FFFFFF;
+
+    /*remotion-themed*/
+    --remotion-themed-border: thin solid var(--border);
+    --remotion-themed-border-focus: thin solid var(--focus);
+    --remotion-themed-border-radius: 0.125em;
+    --remotion-themed-border-hover: thin solid var(--primary);
+    --remotion-themed-dropdown-scrollbar-border-radius: 0.6250em;
+    --remotion-themed-dropdown-scrollbar-width: 0.6250em;
+    --remotion-themed-scrollbar-width: 0.8750em;
+}
+
+input.remotion-themed,
+textarea.remotion-themed,
+select.remotion-themed,
+.DropDownMenuSelect.remotion-themed,
+.remotion-themed input,
+.remotion-themed textarea,
+.remotion-themed select,
+.remotion-themed .DropDownMenuSelect {
+    border: var(--remotion-themed-border);
+    border-radius: var(--remotion-themed-border-radius);
+}
+
+input.remotion-themed:focus,
+textarea.remotion-themed:focus,
+select.remotion-themed:focus,
+.DropDownMenuSelect.remotion-themed:focus,
+.remotion-themed input:focus,
+.remotion-themed textarea:focus,
+.remotion-themed select:focus,
+.remotion-themed .DropDownMenuSelect:focus {
+    border: var(--remotion-themed-border-focus) !important;
+    outline-style: none;
+}
+
+input.remotion-themed:hover,
+textarea.remotion-themed:hover,
+select.remotion-themed:hover,
+.DropDownMenuSelect.remotion-themed:hover,
+.remotion-themed input:hover,
+.remotion-themed textarea:hover,
+.remotion-themed select:hover,
+.remotion-themed .DropDownMenuSelect:hover {
+    border: var(--remotion-themed-border-hover);
+    outline-style: none;
+}
+
+input.remotion-themed:disabled,
+textarea.remotion-themed:disabled,
+select.remotion-themed:disabled,
+.DropDownMenuSelect.remotion-themed:disabled,
+.remotion-themed input:disabled,
+.remotion-themed textarea:disabled,
+.remotion-themed select:disabled,
+.remotion-themed .DropDownMenuSelect:disabled {
+    border: var(--remotion-themed-border);
+    outline-style: none;
+}
+
+a.remotion-themed,
+.remotion-themed a {
+    color: var(--primary);
+    text-decoration: none;
+}
+
+a.remotion-themed:hover,
+.remotion-themed a:hover {
+    color: var(--primary) !important;
+    text-decoration: none !important;
```

Comment:
From one of the screenshots it looks like the a tag should have a underline if hovered.

Reply:
this will be itneresting anyway and require a second roundtrip. please create an RM-ticket linked as follow-up with the questions:
* hover-style of anchor: underline?
* focus-style of anchor: underline or outline?

Also, we should never have to rely on `!important` in our styles.